### ISSUE TITLE
question about data construction

### ISSUE THREAD

**Issue by AIRobotZhang** [graytowne/caser_pytorch#14]
Hello, I want know if removing the user-item rating less than 3 (only remaining the ratings more than 3) in MoiveLen-1M dataset when constructing the dataset. Thank you!

**Comment by miferroudjene** [comment]
Hello there! When I tried it on others datasets (i.e. ML1M-50, Epinions ...), during the preprocessing phase, it made sense to me to delete users with less than 3 products in their purchase history. According to the CASER original paper, they split the data as follows: 

- 70% for training set
- 10% Validation set
- 20% for test set
And, logically, I assume we need at least two items in the training set to learn the sequences. ( correct me if I'm wrong, that's was my point of view )

**Comment by graytowne** [comment]
Hi there, thanks for your interest in our paper! In MovieLens-1M dataset, we recurrently removed users and items that has less than 5 interactions. And we convert the numeric rating score to implicit feedback without a threshold (meaning that whenever there's a score, then it is considered as a positive implicit feedback), you could also use a threshold for that but that will further sparsify the dataset.

Hopefully this answers your questions, I'm closing the issue now. LMK if you have any further questions.
